static/js/main.js

```diff
@@ -625,6 +642,39 @@ $(function() {
     document.location.reload(true);
   });
 
+  function makeShortBorsMessage(commit) {
+    /* Extract out the lines that are generated by bors as the
+    real commit message. Then return these with a '; ' delimimiter.
```

Comment:
I think it would be useful to include an example bors commit message here.

Reply:
Added that.